### PR TITLE
Correction in the formula for mean squared error

### PR DESCRIPTION
**Pull request by krishnaw14**
There was a minor error in the formula for mean squared error in neural_nets.ipynb

### CODE REVIEW

File: neural_nets.ipynb [hunk 1]
```diff
@@ -82,7 +82,7 @@
     "\n",
     "In both the Perceptron and the Neural Network, we are using the Backpropagation algorithm to train our weights. Basically it achieves that by propagating the errors from our last layer into our first layer, this is why it is called Backpropagation. In order to use Backpropagation, we need a cost function. This function is responsible for indicating how good our neural network is for a given example. One common cost function is the *Mean Squared Error* (MSE). This cost function has the following format:\n",
     "\n",
-    "$$MSE=\\frac{1}{2} \\sum_{i=1}^{n}(y - \\hat{y})^{2}$$\n",
+    "$$MSE=\\frac{1}{2n} \\sum_{i=1}^{n}(y - \\hat{y})^{2}$$\n",
```
Comment (norvig) [inline neural_nets.ipynb]: should be \frac{1}{n}

Comment (krishnaw14) [reply neural_nets.ipynb]: I guess both conventions are used. In [this](http://neuralnetworksanddeeplearning.com/chap2.html#the_two_assumptions_we_need_about_the_cost_function) website, it uses 2n and in [this](https://isaacchanghau.github.io/2017/06/07/Loss-Functions-in-Artificial-Neural-Networks/), it uses n. 

Anyways, I made the changes.

**Comment by norvig** [comment]
Oops, there are conflicts agin, due to another PR. These are really hard to review for .ipynb

**Comment by krishnaw14** [comment]
Should I send another PR after pulling the latest changes from the repository? @norvig

**Comment by ad71** [comment]
This PR contains duplicated changes. It will be easier if you close this PR, update your working directory, create a new branch and open a fresh pull request. Always make a branch before you commit changes. The master branch is for updating the local directory from the upstream repo.

**Pull request closed without merging**